src/util.py

```diff
@@ -11,4 +12,27 @@ def update_config(config,args,exclude=[],required=[]):
         if (config.get(param) is None):
             raise ValueError("parameter {0} unspecified. Please provide a value via the command line or in the config file.".format(param))
 
-    return config
\ No newline at end of file
+    return config
+    
+def recursively_find_files(resource_name):
```

Comment:
@plauto I moved your method into utils because it's quite general purpose. also there was a small bug in case it was passed a file, should be fixed now. No action required just a comment